src/core/event/createGetModifierState.js

```diff
@@ -0,0 +1,19 @@
+const modifierEventKeyMap = {
+  Alt: 'altKey',
+  Control: 'ctrlKey',
+  Meta: 'metaKey',
+  Shift: 'shiftKey'
+};
+
+export default function createGetModifierState(rawEvent) {
```

Comment:
Is there a way to do this without a closure? Not a huge thing, but it seems like an ES5 idiom. Even the `createGetModifierState` name is a good indication, combining `create` and `get` in the same function makes it sound more complex than it is.